### PR TITLE
add API_LEVEL infos in Readme

### PR DESCRIPTION
**Pull request by sgliner-ledger**
## Description

Add documentation

## Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)
- [ ] Tests
- [X] Documentation
- [ ] Other (for changes that might not fit in any category)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,3 +71,16 @@ To enable pre-commit in your development environment:
...
     ```
+## About API_LEVEL branches
+
+| Name    | Related OS                                                                                                    | Active
+|---------|---------------------------------------------------------------------------------------------------------------|---------
+| 1       | nanox_2.1.0 <br/>  nanos+_1.1.0 <br/>                                                                         | :heavy_check_mark:
+| 2       | nanox_2.2.0-pr1 <br/>  nanos+_1.2.0-pr1 <br/>                                                                 | :heavy_check_mark:
+| 3       | stax_1.0.0-rc8                                                                                                | :x:
+| 4       | nanox_2.2.0-pr2 <br/> nanos+_1.2.0-pr2 <br/>                                                                  | :x:
+| 5       | stax_1.0.0-rc9 <br/> nanox_2.2.0-rc1 <br/> nanos+_1.2.0-rc1 <br/> nanox_2.2.0-pr3 <br/> nanox_2.2.0-rc2 <br/> | :heavy_check_mark:
```
Comment (xchapron-ledger) [inline README.md:82]: can you add `nanox_2.2.0` here too?

File: README.md [hunk 2]
```diff
@@ -71,3 +71,16 @@ To enable pre-commit in your development environment:
     pre-commit install --hook-type pre-commit
     pre-commit install --hook-type commit-msg
     ```
+## About API_LEVEL branches
```
Comment (xchapron-ledger) [inline README.md:74]: To be put above contributiong section?

Comment (sgliner-ledger) [reply README.md:74]: :shrug: If you want :smile:

File: README.md [hunk 3]
```diff
@@ -51,6 +51,34 @@ In short, to build an app for an OS, you should:
...
+
+- Fetch last changes from remote: `git fetch --all`
+
+- Create a new branch to hold your cherry-picks: `git chechout origin/API_LEVEL_X -b mybranch`
+
+- Cherry-pick your commits: `git cherry-pick -x commit_sha1` (the -x is useful to track the original commit of a cherry-pick).
+
+- Push your branch: `git push origin mybranch`
+
+- Create a PR and indicate in it the PR where your cherry-pick where reviewed first.p
```
Comment (xchapron-ledger) [inline README.md:80]:
```suggestion
- Create a PR and indicate in it the PR where your cherry-pick where reviewed first.

```

**Review by xchapron-ledger — APPROVED** [review]

**Pull request merged**